account_move_import_queue_context/tests/test_account_move_import_queue_context.py

```diff
@@ -0,0 +1,12 @@
+from odoo.tests.common import TransactionCase
+
+
+class TestJobPrepareContext(TransactionCase):
+    def test_job_prepare_context_before_enqueue_keys(self):
+        model = self.env["base"].create({})
```

Comment:
**question**(non-blocking): do you need this `create`? Couldn't you just do `self.env["base"]._job_prepare_context_before_enqueue_keys()` ?

Reply:
I concur the create is unnecessary for methods decorated `@api.model`.